modules/exploits/multi/http/gitlist_arg_injection.rb

```diff
@@ -0,0 +1,65 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = NormalRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'           => "GitList v0.6 Argument Injection",
```

Comment:
Usually we leave the version out of the title and document it in the description, target(s), and doc.